.github/workflows/release.yaml

```diff
@@ -0,0 +1,52 @@
+name: 'Release'
+ # Version bump and release
+ on:
+   workflow_run:
+     workflows: ["Validate"]
+     type:
+       - complete
+
+ jobs:
+   bump-version:
+     name: 'Package, Bump Version and Tag'
+     runs-on: ubuntu-latest
+     if: ${{ !contains(github.event.head_commit.message, 'skip ci') && github.event.workflow_run.conclusion == 'success' }}
+
+     steps:
+       - name: 'Checkout source code'
+         uses: 'actions/checkout@v2'
+         with:
+           ref: ${{ github.ref }}
+
+       - name: Set Node.js 12.x
```

Comment:
Is there any reason to not use the latest LTS version?

Reply:
No really. Official actions somehow are using 12, but we can use 16. I've updated the PR.

Reply:
@mpermar Did you update it?

Reply:
I forgot there were two workflows. It's updated now.